SQF/dayz_server/compile/server_publishFullObject.sqf

```diff
@@ -12,7 +12,9 @@ _fuel =			_this select 7;
 _allowed = [_object, "Server"] call check_publishobject;
 if (!_allowed) exitWith { deleteVehicle _object; };
 
```

Comment:
You need to add this include at the top of files that don't already have it or the macros won't work:
```sqf
#include "\z\addons\dayz_server\compile\server_toggle_debug.hpp"
```